dstep/translator/CommentIndex.d

```diff
@@ -68,6 +68,73 @@ class CommentIndex
         {
             return offset < s ? -1 : (offset == s ? 0 : 1);
         }
+
+        private static bool isNormalized(string content)
+        {
+            import std.ascii : isWhite;
+
+            for (size_t i = 1; i < content.length; ++i)
+            {
+                if (content[i] == '\n' && content[i - 1].isWhite)
+                    return false;
+            }
```

Comment:
I do not see how. There is three versions of `canFind`. The versions with `needle` cannot be used as there is no fixed sub-string to search for (rather a sequence of white-space+new-line). And the predicate considers only one element at once and I need to consider two subsequent elements.